content/posts/why-sre/index.mdx

```diff
@@ -0,0 +1,163 @@
+ <!-- Title, author, date block standard in all blog post files -->
+# Why SRE?
+*Author(s):@david-martin*<br/>
+<!-- Hide this human-readable detail so content doesn't appear to be stale or dated
+*Date Published: 2022-05-16*<br/>
```

Comment:
please add the metadata with frontmatter like https://github.com/operate-first/blog/blame/ff217f9d7dba37bcd14b044829cdfd41a1f37632/content/posts/hello-world/index.mdx#L1-L6